egret/model_library/unit_commitment/params.py

```diff
@@ -194,28 +194,26 @@ def load_params(model, model_data, slack_type):
     # penalty costs for constraint violation #
     ##########################################
 
```

Comment:
General comment: these are sufficiently important constants that I think they warrant a big comment block in the corresponding files, defining the values and the hierarchy/presence.